assets/stylesheets/common/common.scss

```diff
@@ -1366,3 +1366,17 @@ body.has-full-page-chat {
     white-space: nowrap;
   }
 }
+
+.chat-flag-inputs {
```

Comment:
Does this element exist? Can't find it from a quick grep of the branch, but maybe I'm doing it wrong.